recipes/freebayes/meta.yaml

```diff
@@ -1,16 +1,15 @@
 package:
   name: freebayes
-  version: '1.1.0.46'
+  version: '1.2.0'
 
 source:
   git_url: https://github.com/ekg/freebayes.git
-  git_tag: 8d2b3a060da473e1f4f89be04edfce5cba63f1d3
+  git_tag: 40155b407a4bead708aaafcadf7904854c411275
```

Comment:
It would be good to use the official release tarball next time.

Reply:
I was planning of using the tarball @bgruening , but "freebayes" has few submodule dependencies that are not part of the official release. I plan to make recipes for the missing dependencies, but that may take some time.